e2e_tests/tests/deploy/test_local.py

```diff
@@ -254,6 +267,31 @@ def test_master_up_down() -> None:
     assert len(containers) == 0
 
 
+@pytest.mark.det_deploy_local
+def test_master_up_interactive_password() -> None:
+    cluster_name = "test_master_up_interactive_password"
+    master_name = f"{cluster_name}_determined-master_1"
+    password_input = "XDdOB9VUp8FLpTZ2"
+
+    with resource_manager(
+        Resource.MASTER,
+        master_name,
+        boolean_flags=["delete-db"],
+        cmd_input=bytes(f"{password_input}\n{password_input}\n", "utf8"),
+    ) as master_up_command:
+        client = docker.from_env()
+
+        containers = client.containers.list(filters={"name": master_name})
+        assert len(containers) > 0
+        assert (
+            b"Determined Master was launched without a strong initial_user_password set."
+            in master_up_command.stdout
+        )
+
+    containers = client.containers.list(filters={"name": master_name})
+    assert len(containers) == 0
```

Comment:
can we run this with multiple cases? of passwords that pass & those that don't & and empty case?